com.weibellab.comms/Runtime/ReliableCommunication.cs

```diff
@@ -307,6 +319,123 @@ public void StartConnection()
             }
         }
 
+        /// <summary>
+        /// Sets the target IP and Port based on set strategy
+        /// By default, the user manually sets the port and IP.
+        /// Optionally, they can ping a web server to get other device info.
+        /// </summary>
+        private async void setTargetAndStartConnection()
+        {
```

Comment:
Added an intermediary function to the ReliableCommunication startup process. This function is basically just checking to see if it should ping the webserver or use the manually input ip/ports.